docs/api/session.md

```diff
@@ -181,15 +181,13 @@ Clears the data of web storages.
 
 #### `ses.setProxy(config, callback)`
 
-* `config` String
+* `config` Object
+  * `pacScript` String - The URL associated with the PAC file.
+  * `proxyRules` String - Rules indicating which proxies to use.
```

Comment:
[proxy_config.h](https://code.google.com/p/chromium/codesearch#chromium/src/net/proxy/proxy_config.h&sq=package:chromium) contains the following comment:
```C++
// There are two categories of proxy settings:
//   (1) Automatic (indicates the methods to obtain a PAC script)
//   (2) Manual (simple set of proxy servers per scheme, and bypass patterns)
```
So maybe we should change `pacScript` to `automatic` and `proxyRules` to `manual` for consistently.